src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -10,6 +10,12 @@ public class CliSyntax {
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
+    public static final Prefix PREFIX_CLAZZ = new Prefix("c/");
+    public static final Prefix PREFIX_PERSONALITY = new Prefix("P/");
+    public static final Prefix PREFIX_ATTENDANCE = new Prefix("A/");
+    public static final Prefix PREFIX_SUBJECT = new Prefix("s/");
+    public static final Prefix PREFIX_GRADE = new Prefix("g/");
```

Comment:
Consider changing the prefix to something less confusing, e.g. "pr/" for personality and "att/" for attendance